<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodeWidget</class>
 <widget class="QWidget" name="EncodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="alphabet_file_dialog_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
  <widget class="QPushButton" name="encode_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>630</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Закодировать</string>
   </property>
  </widget>
  <widget class="QLabel" name="encode_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Кодирование</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="encode_seq_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="encode_seq_file_dialog_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="alphabet_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="codesTextBrowser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>371</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="alphabet_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Путь к файлу с алфавитом</string>
   </property>
  </widget>
  <widget class="QLabel" name="sencode_seq_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пусть к файлу с последовательностью для кодирования</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="encodedTextEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>371</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="changePushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>540</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QLabel" name="symbolsAndCodeslabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Символы и их коды</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="encodedLabel">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>280</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Закодированная последовательность</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
